python/orca/test/bigdl/orca/learn/ray/tf/test_tf2estimator_ray_dataset.py

```diff
@@ -109,6 +109,9 @@ def test_train_and_evaluate(self):
                                                  label_cols="x", feature_cols=["y"],
                                                  data_config=data_config_args)
 
+        assert type(train_stats) == dict
+        assert type(end_eval_stats) == dict
+
```

Comment:
Also need to add assert for other input types (e.g. Spark DataFrames, SparkXShards, etc.)